cmd/system-probe/main_common.go

```diff
@@ -23,7 +24,7 @@ func setDefaultCommandIfNonePresent(rootCmd *cobra.Command) {
 	args := []string{os.Args[0], "run"}
 	if len(os.Args) > 1 {
 		potentialCommand := os.Args[1]
-		if potentialCommand == "help" {
+		if potentialCommand == "help" || potentialCommand == "completion" {
```

Comment:
There's a flag to disable the `completion` command which we can set on the system-probe command:
`sysprobeCmd.CompletionOptions.DisableDefaultCmd = true`

Reply:
I'm OK with having it, if folks really want it.